src/notation/qml/MuseScore/NotationScene/internal/PartDelegate.qml

```diff
@@ -160,8 +161,11 @@ ListItemBlank {
 
         MenuButton {
             Component.onCompleted: {
-                var operations = [ { "id": "duplicate", "title": qsTrc("notation", "Duplicate") },
-                                   { "id": "rename", "title": qsTrc("notation", "Rename") }]
+                var operations = [
+                            { "id": "duplicate", "title": qsTrc("notation", "Duplicate") },
+                            { "id": "rename", "title": qsTrc("notation", "Rename") },
+                            { "id": "reset", "title": qsTrc("notation", "Reset"), "enabled": root.isInited }
+                        ]
 
                 if (root.isCustom) {
```

Comment:
As far as I can tell, this code will no longer work because you renamed the property

Reply:
Thanks, I forgot to add a new `isCustom` property. Now I named it `canDelete`.